docs/llm/quickstart.md

```diff
@@ -10,16 +11,17 @@ Requirements:
 
 1. Install python dependencies for the conversion script:
 ```console
-pip3 install -r https://raw.githubusercontent.com/openvinotoolkit/model_server/refs/heads/main/demos/common/export_models/requirements.txt
+pip3 install huggingface_hub
 ```
 
 2. Run optimum-cli to download and quantize the model:
 ```console
-curl https://raw.githubusercontent.com/openvinotoolkit/model_server/refs/heads/releases/2025/1/demos/common/export_models/export_model.py -o export_model.py
+curl https://raw.githubusercontent.com/openvinotoolkit/model_server/refs/heads/simpler-quick-start-llm/demos/common/export_models/export_model.py -o export_model.py
```

Comment:
```suggestion
curl https://raw.githubusercontent.com/openvinotoolkit/model_server/refs/heads/main/demos/common/export_models/export_model.py -o export_model.py
```

Reply:
@dtrawins shall we revert it before merge?